plugins/Scheduler/plugin.py

```diff
@@ -90,7 +90,12 @@ def _restoreEvents(self, irc):
                         # though we'll never know for sure, because other
                         # plugins can schedule stuff, too.
                         n = int(name)
-                    self._add(network, event['msg'], event['time'], event['command'], n)
+                    is_reminder = event.get('is_reminder', False)
+                    # Here we use event.get() method instead of event[]
+                    # This is to maintain compatibility with older bots
+                    # lacking 'is_reminder' in their event dicts
```

Comment:
swap the comment and the definition of `is_reminder` (it's more common to write comments before the code they are describing)